cmd/algoh/main.go

```diff
@@ -362,7 +362,7 @@ func captureErrorLogs(algohConfig algoh.HostConfig, errorOutput stdCollector, ou
 
 func reportErrorf(format string, args ...interface{}) {
 	fmt.Fprintf(os.Stderr, format, args...)
-	logging.Base().Fatalf(format, args...)
+	logging.Base().Warnf(format, args...)
```

Comment:
Would Errorf be better than Warnf?

Reply:
hmm. The thing is this - 
Errorf ends up writing a stack trace dump into the log file. This stack trace is completely useless, since it's the `algoh` stack trace. It also doesn't indicate an error in the `algoh`, since the `algoh` was working flawlessly.

Reply:
also the algoh hasn't errored, the algod has right?